recipes/arrow/all/conandata.yml

```diff
@@ -48,24 +48,83 @@ sources:
+  "17.0.0":
+    - patch_file: "patches/16.0.0-0001-fix-cmake.patch"
+      patch_description: "use cci package"
+      patch_type: "conan"
+  "16.1.0":
+    - patch_file: "patches/16.0.0-0001-fix-cmake.patch"
+      patch_description: "use cci package"
+      patch_type: "conan"
+  "16.0.0":
+    - patch_file: "patches/16.0.0-0001-fix-cmake.patch"
+      patch_description: "use cci package"
+      patch_type: "conan"
+  "15.0.0":
+    - patch_file: "patches/11.0.0-0001-fix-cmake.patch"
+      patch_description: "use cci package"
+      patch_type: "conan"
+  "14.0.2":
+    - patch_file: "patches/11.0.0-0001-fix-cmake.patch"
+      patch_description: "use cci package"
+      patch_type: "conan"
+  "14.0.1":
+    - patch_file: "patches/11.0.0-0001-fix-cmake.patch"
+      patch_description: "use cci package"
+      patch_type: "conan"
+  "14.0.0":
+    - patch_file: "patches/11.0.0-0001-fix-cmake.patch"
+      patch_description: "use cci package"
+      patch_type: "conan"
+  "13.0.0":
+    - patch_file: "patches/11.0.0-0001-fix-cmake.patch"
+      patch_description: "use cci package"
+      patch_type: "conan"
+  "12.0.1":
+    - patch_file: "patches/11.0.0-0001-fix-cmake.patch"
+      patch_description: "use cci package"
+      patch_type: "conan"
+  "12.0.0":
+    - patch_file: "patches/11.0.0-0001-fix-cmake.patch"
+      patch_description: "use cci package"
+      patch_type: "conan"
+  "11.0.0":
+    - patch_file: "patches/11.0.0-0001-fix-cmake.patch"
+      patch_description: "use cci package"
+      patch_type: "conan"
   "8.0.1":
-    - patch_file: "patches/8.0.0-0005-install-utils.patch"
+    - patch_file: "patches/8.0.0-0001-static-analyzers.patch"
+      patch_description: "do not look for static analyzers"
+      patch_type: "conan"
+    - patch_file: "patches/8.0.0-0002-install-utils.patch"
       patch_description: "enable utils installation"
       patch_type: "conan"
-    - patch_file: "patches/8.0.0-0006-fix-cmake.patch"
+    - patch_file: "patches/8.0.0-0003-fix-cmake.patch"
       patch_description: "use cci package"
       patch_type: "conan"
+    - patch_file: "patches/8.0.0-0004-fix-msvc-build.patch"
+      patch_description: "see commit a53f2bda5dfa50f2d1dbfd9d7fbc98d8c9725564 from arrow's repo"
+      patch_type: "portability"
   "8.0.0":
-    - patch_file: "patches/8.0.0-0005-install-utils.patch"
+    - patch_file: "patches/8.0.0-0001-static-analyzers.patch"
+      patch_description: "do not look for static analyzers"
+      patch_type: "conan"
+    - patch_file: "patches/8.0.0-0002-install-utils.patch"
       patch_description: "enable utils installation"
       patch_type: "conan"
-    - patch_file: "patches/8.0.0-0006-fix-cmake.patch"
+    - patch_file: "patches/8.0.0-0003-fix-cmake.patch"
       patch_description: "use cci package"
       patch_type: "conan"
+    - patch_file: "patches/8.0.0-0004-fix-msvc-build.patch"
+      patch_description: "see commit a53f2bda5dfa50f2d1dbfd9d7fbc98d8c9725564 from arrow's repo"
```

Comment:
Maybe add a more descriptive description and a `patch_source` to the commit.
Or preferably the PR where this were introduced